docs/standards/react.md

```diff
@@ -0,0 +1,170 @@
+# React Standards
+
+## Controller-View Pattern
+
+Every feature follows a controller-view-hook pattern:
+
+```
+feature/
+  feature.tsx            # Orchestration
+  feature.hook.ts        # Business logic & state
+  feature.hook.spec.ts   # Hook tests
+  feature.view.tsx       # Pure presentation
+  feature.view.spec.tsx  # View tests
+  feature.md             # Documentation (optional)
```

Comment:
nice. 💯 

Some have a tendency of making every markdown file capital, thinking it's a convention when they see README, while it's only to say reaaally read this.

```suggestion
  feature.md             # Documentation (optional), yes, lowercase file name
```